subjects/zappy/README.md

```diff
@@ -0,0 +1,368 @@
+
+```console
+
+              ---- ---- ----
+             | 1  | 2  | 3  |
+              ---- ---- ----
+                  | 0  |
+                   ----
+                  Level 1
+
+         ---- ---- ---- ---- ----
+        | 4  |  5 |  6 |  7 |  8 |
+         ---- ---- ---- ---- ----
+             | 1  | 2  | 3  |
+              ---- ---- ----
+                  | 0  |
+                   ----
+                  Level 2
+
+ ----- ----- ----- ----- ----- ----- -----
+|  9  | 10  | 11  | 12  | 13  | 14  | 15  |
+ ----- ----- ----- ----- ----- ----- -----
+      |  4  |  5  |  6  |  7  |  8  |
+       ----- ----- ----- ----- -----
+             | 1  | 2  | 3  |
+              ---- ---- ----
+                  | 0  |
+                   ----
+                  Level 3
+
+```
+
+For the player to know his surroundings, the `client` sends the command `SEE` and the server responds the following chain of characters (for level 1):
+
+```console
+{content-square-0, content-square-1, content-square-2, content-square-3}
+```
+
+Our player doesn't even see himself, and if there is more than 1 object in a box, they are all indicated and separated by a space:
+
+```console
+              -------- -------- --------
+             | sibur  | Phiras |        |
+             | player | Phiras |        |
+              -------- -------- --------
+                      |        |
+                      |  food  |
+                       --------
+                        Level 1
+
+```
+
+```console
+{food, player sibur, phiras phiras, }
+```
+
+### The Time and Game Speed
+
+The game speed is defined by a time unit. Each action in the game has a duration proportionate to this time unit.
+
+The time unit is defined by the function 1/t :
+
+- t is passed as parameter at the start of the server.
+- A time unit lasts 1/t seconds.
+
+- No active wait time will be tolerated.
+- There cannot be any blocking when the clients are stopped, or in any phase of the game.
+- The Trantorians have adopted an international time unit.
+- Time unit is a second. If `t=1` “forward” takes 7 seconds. We choose by default, t=100. t is a integer.
+- The time reference is absolute time.
+
+### Objects denomination
+
+Only the classification of an object can be identifiable. It is therefore impossible to distinguish two objects of the same class. For example, two `siburs` will have the same denomination since they belong to the same class.
+
+### Teams
+
+At the beginning a team is made of n player and only n. Each player is controlled by a client. The clients cannot communicate or exchange amongst each other data outside of the game, in any way.
+
+At the beginning the client has 10 life units, he can therefore survive 1260 time units, 1260/t seconds.
```

Comment:
done

Reply:
not rephrased. As an example of rephrasing:
"Each player starts with 10 `food`."

Rational behind:
- Time has already his own section, so I don't feel useful to repeat the time logic here
- `food` is a specific game item, so i feel it should be marked `food ` instead of food
- The concept of `client` can be confusing and so I try to help the student by saying `player`

Further possible improvements:
- Why is that information in the `Teams` section? Maybe it will be better moved into the `Player` section, which leaves `Team` with only one unclear point that needs rephrasing, so maybe there is room to remove completely the `Team` session or rethink it to make it more useful.

When I ask for rephrasing I feel there is a need or a possibility to improve and make it more clear or to introduce better notions, it is not only about replacing words with synonyms, so it should imply conscious thoughts.